src/modules/RegAuth/Auth/components/Auth.tsx

```diff
@@ -0,0 +1,113 @@
+import cn from 'classnames';
+import React from 'react';
+import {Trans} from 'react-i18next';
+import {NavLink} from 'react-router-dom';
+import {Field} from 'redux-form';
+
+import {socialNetworks} from 'modules/RegAuth/common/constants/constants';
+
+import Flex from 'common/components/Flex/Flex';
+
+import {COLORS} from 'common/constants/entities';
+import rules from 'modules/RegAuth/Registration/validation/InputValidation';
+
+import SuppliesIcon from 'assets/images/SuppliesIcon';
+
+import InputWithValidation from 'modules/RegAuth/InputWithValidation/InputWithValidation';
+
+import silentClasses from 'styles/_silent_classes.scss';
+import styles from './auth.scss';
+
+const Auth = (props: any) => (
+	<Flex
+		className={styles.wrapper}
+	>
+		<Flex
+			alignItems='center'
+			direction='column'
+			className={styles.container}
+		>
+			<Flex
+				direction='column'
+				alignItems='center'
+				className={styles.header}
+			>
+				<Flex indent='medium'>
+					<SuppliesIcon className={styles.cusdebIcon} fill={COLORS.green800} />
+				</Flex>
+				<Flex indent='medium' className={silentClasses.toUpperCase}>
+					<Trans i18nKey='RegAuth.Auth.SignInToCusdeb' />
+				</Flex>
+				<Flex indent='medium' className={styles.dontHaveAccount}>
+					<Flex indent='small'>
+						<Trans i18nKey='RegAuth.Auth.DontHaveAnAccount' />?
+					</Flex>
+					<Flex indent='small'>
+						<NavLink to='/registration'>
+							<Trans i18nKey='RegAuth.Auth.Register' />
+						</NavLink>
+					</Flex>
+				</Flex>
+			</Flex>
+			<Flex
+				indent='large'
+				justifyContent='center'
+				alignItems='center'
+				className={styles.resetPwdWrapper}
+			>
+				<Flex
+					className={styles.resetPwdContainer}
+				>
+					<Flex alignItems='center'>
+						<Flex indent='small'><Trans i18nKey='RegAuth.Auth.ForgotPassword' /></Flex>?
+						<Flex indent='small'>
+							<NavLink to='/reset'><Trans i18nKey='common.ClickHere' /></NavLink>
+						</Flex>
+					</Flex>
+				</Flex>
+			</Flex>
+			<Flex
+				className={styles.iconsBlock}
+				alignItems='center'
+				justifyContent='center'
+				indent='large'
+			>
+				{socialNetworks(50).map(item => (
+					<Flex indent='small' className={styles.icon}>
```

Comment:
That mapped items should have keys to avoid that:
![Image of Error](https://cdn.discordapp.com/attachments/418795190061629441/543422652484943882/Screenshot_from_2019-02-08_16-27-27.png)

Reply:
Same deal on Auth page